megamek/src/megamek/common/MiscType.java

```diff
@@ -1383,6 +1383,7 @@ public int getCriticals(Entity entity, double size) {
             return (int) Math.ceil(size);
         }
         // right, well I'll just guess then
+        LOGGER.error("Failed to calculate crit slots for equipment %s".formatted(this.getName()));
```

Comment:
Replace with 

`LOGGER.error("Failed to calculate crit slots for equipment {}", this.getName());`

Reply:
Done